comms/opencbm/Makefile

```diff
@@ -0,0 +1,74 @@
+# Created by: Felix Palmen <felix@palmen-it.de>
+# $FreeBSD$
+
+PORTNAME=	opencbm
+PORTVERSION=	${OCBM_VERSION}
+CATEGORIES=	comms archivers
+
+PATCHFILES=	12ef17d4cf78903c9c52d7a348a816424d549b89.diff
+
+MAINTAINER=	felix@palmen-it.de
+COMMENT=	Commodore CBM serial IEC/IEEE-488 bus utilities
+
+LICENSE=	GPLv2 BSD3CLAUSE
+LICENSE_COMB=	multi
+
+BUILD_DEPENDS=	cc65:devel/cc65
+
+USES=		gmake
+USE_LDCONFIG=	yes
+
+OPTIONS_DEFINE=		DOCS INFO
+OPTIONS_DEFAULT=	DOCS INFO
+
+MAKE_ARGS=	CC=cc INFODIR=${PREFIX}/share/info
+MAKEFILE=	LINUX/Makefile
+MAKE_JOBS_UNSAFE=	yes
+ALL_TARGET=	opencbm plugin-xu1541 plugin-xum1541
+INSTALL_TARGET=	install install-plugin-xu1541 install-plugin-xum1541
+
+DOCS_BUILD_DEPENDS=	dvips:print/tex-dvipsk \
+			latex:print/tex-formats \
+			linuxdoc:textproc/linuxdoc-tools
+DOCS_USES=		ghostscript:build
+DOCS_ALL_TARGET=	doc
+DOCS_MAKE_ARGS_OFF=	TXT= DVI= PDF= HTML=
+
+INFO_BUILD_DEPENDS=	linuxdoc:textproc/linuxdoc-tools
+INFO_USES=		makeinfo
+INFO_ALL_TARGET=	doc
+INFO_INFO=		opencbm
+INFO_MAKE_ARGS_OFF=	INFO=
+
+OCBM_CONFDIR=	${LOCALBASE}/etc/opencbm.conf.d
+
+post-patch:
+	@${REINPLACE_CMD} -e 's|_ETC="$$1"|_ETC="$${1:-${LOCALBASE}/etc}"|' \
+		-e 's|in /.*/ instead|in ${OCBM_CONFDIR}/ instead|' \
+		-e "s|execute 'opencbm_|execute '|" \
+		${WRKSRC}/opencbm/LINUX/plugin_helper_tools
+
+post-install:
+	${STRIP_CMD} ${STAGEDIR}${PREFIX}/lib/libopencbm.so.${OCBM_LIBVERSION}
+.for p in xu1541 xum1541
+	${STRIP_CMD} ${STAGEDIR}${PLUGINBASENAME}${p}.so.${OCBM_LIBVERSION}
```

Comment:
You can use glob
${STRIP_CMD} ${STAGEDIR}${PLUGINBASENAME}*.so.${OCBM_LIBVERSION}
You can "merge" the two strip lines

Reply:
That's a good idea! I now submitted a second PR to fix the "real" issues found, and will keep this for the next update. Thanks again :)